mocked-api/schema_types/blood_donation_center.py

```diff
@@ -3,39 +3,97 @@
 from typing import Optional
 from strawberry import relay
 from datetime import datetime
-from strawberry.types import Info
-from typing import Iterable
+
+# from strawberry.types import Info
+# from typing import Iterable
 from schema_types.blood_dotation_center_category import (
-    ENUM_BLOODDONATIONCENTER_CATEGORY,
+    BloodCenterCategory,
 )  # noqa: E501
+
 from schema_types.tag import Tag
 from schema_types.address import Address
 from schema_types.contacts import Contacts
 from schema_types.work_schedule_item import WorkScheduleItem
 
-# from mocked.mocked_blood_dotation_centers import mocked_data
-
 
 @strawberry.type
 class BloodDonationCenter(relay.Node):
+    attributes: strawberry.Private[object]
```

Comment:
Could you have defined attributes as also a relay.Node?
It may be that strawberry does some validations/type checks for you, defining each field then separately will cause you to lose these benefits.

Reply:
Inheriting from relay.Node makes sense in specific scenarios. This can be beneficial if we need to fetch BloodDonationCenterAttributes directly using a unique ID. But it is not our case.

Reply:
Alright